falafel/core/evaluators.py

```diff
@@ -335,15 +335,22 @@ def get_contextual_hostname(self, default=""):
         return self.system_id
 
     def get_branch_info(self):
+        version = None;
+        hostname = None;
```

Comment:
@jeudy100 you seem to have injected a bit of Java here, semicolons are not required.  If you fix this and rebase your branch it will fix the errors in CI.  You can confirm this by running `flake8` in the project root directory.